### ISSUE TITLE
Links may not have proper focus outline/ring as of Chromium/Chrome/Edge 83+

### ISSUE THREAD

**Issue by michael-milette** [wet-boew/wet-boew#8873]
On May 23rd, 2020, Google launched Chrome 83 which includes changes to the default styles we have all known for years, especially for links and form elements.

Unfortunately this has had a negative impact on the WET-BOEW project. For example, the focus outline no longer displays correctly in breadcrumbs on Government of Canada Websites with the GCWeb theme:

![image](https://user-images.githubusercontent.com/3808579/83394497-b42e3280-a3c6-11ea-945d-8c8c3862ebac.png)

It seems that the new thicker black outline no longer fits in the tight space permitted in the GCWeb theme's breadcrumb items.

The following link mentions the change in the outline:
* https://developers.google.com/web/updates/2020/05/nic83#forms
* https://blog.chromium.org/2020/03/updates-to-form-controls-and-focus.html

You can still go back and see what it used to look like using the following experimental settings, but only until Chrome 84:
chrome://flags/#form-controls-refresh

Time to check all your links and UI throughout your websites folks!

Best regards,

Michael Milette

**Comment by GormFrank** [comment]
Good catch, but until we find other bugs/issues, we will need to move this issue to GCWeb as the highlight looks fine on the base WET template.
